sass-mixins/nitro-ui/forms/_validation-states.scss

```diff
@@ -53,4 +71,7 @@
 }
 .has-error {
   @include form-control-validation($state-danger-text, $state-danger-text, $state-danger-bg);
+  &-corner {
+    @include form-control-validation-corner($state-danger-text);
```

Comment:
@drborges see https://github.com/powerhome/nitro-web/pull/8283 for the implementation